protocol/testing/genesis.sh

```diff
@@ -61,6 +61,12 @@ function edit_genesis() {
 		INITIAL_CLOB_PAIR_STATUS='STATUS_ACTIVE'
 	fi
 
+	REWARDS_VESTER_ACCOUNT_BALANCE="$7"
```

Comment:
Besides `dev` `local` and `staging` can you make sure https://github.com/dydxprotocol/v4-chain/blob/f50431e3a43a804fc9d4337d779bc054deffd7d9/protocol/scripts/genesis/prod_pregenesis.sh works, and run this [script](https://github.com/dydxprotocol/v4-chain/blob/a7b0d9de142fedfd7f42d05aec6ca395c3ca21f5/protocol/scripts/genesis/check_sample_pregenesis_uptodate.sh) to make sure the sample pregensis file is up-to-date?

Reply:
Updated `prod_pregenesis,sh` as well. When I run that script to check whether it's up-to-date, the content is the same except that parameters have a different ordering, for example:
```
<             "ticker": "ARB-USD",
<             "id": 24,
<             "market_id": 24,
<             "liquidity_tier": 1
---
>             "id": 24,
>             "liquidity_tier": 1,
>             "market_id": 24,
>             "ticker": "ARB-USD"
```

Reply:
Can you rebase from `main` and try again? We recently made a change that fixes non-deterministic ordering in sample pregenesis

Reply:
merged main and ran again
```
Diffing output against current sample_pregenesis.json...
./scripts/genesis/sample_pregenesis.json is up-to-date
```